code/modules/mob/living/carbon/xenomorph/abilities/queen/queen_powers.dm

```diff
@@ -351,9 +353,27 @@
 		target_xeno.evolution_stored += evo_points_per_larva
 
 	user_xeno.hive.stored_larva--
-	return ..()
+	return
+
+/datum/action/xeno_action/onclick/manage_hive/use_ability(atom/Atom)
+	var/mob/living/carbon/xenomorph/queen/queenbanish = owner
 
-/datum/action/xeno_action/onclick/banish/use_ability(atom/Atom)
+
+
+	var/choice = tgui_input_list(queenbanish, "Manage The Hive", "Hive Management",  list("Banish", "Re-Admit", "De-evolve", "Reward Jelly", "Exchange larva for evolution",), theme="hive_status")
+	switch(choice)
+		if("Banish")
+			banish()
+		if("Re-Admit")
+			readmit()
+		if("De-evolve")
+			de_evolve_other()
+		if("Reward Jelly")
+			give_jelly_award(queenbanish.hive)
```

Comment:
Check this because you've removed the plasma cost for this by calling `give_jelly_award()` directly.
https://github.com/cmss13-devs/cmss13/blob/f5300064c56905a93827e9bde911f4853c85868a/code/modules/mob/living/carbon/xenomorph/Abilities.dm#L331-L339

Reply:
I'd prefer the abilities moved into this manage hive ability still check and use plasma if they did before.

Reply:
Agree @Drulikar any chances to plasma costs are out of scope of this PR really.